src/v1/util/secrets.ts

```diff
@@ -0,0 +1,24 @@
+import * as dotenv from 'dotenv';
+import * as fs from 'fs';
+
+if (fs.existsSync('.env')) {
+  console.log('Using .env file to supply config environment variables');
+  dotenv.config({path: '.env'});
+} else {
+  console.log('Using .env.config file to supply config environment variables');
+  dotenv.config({path: '.env.config'}); // you can delete this after you create your own .env file!
+}
+//export const ENVIRONMENT = process.env.NODE_ENV;
+//const prod = ENVIRONMENT === 'production'; // Anything else is treated as 'dev'
+
+export const ADMIN_SYSTEM_KEY = process.env['ADMIN_SYSTEM_KEY'];
```

Comment:
Nitpick so feel free to ignore - instead of exporting the environment variable, in a separate project, I've used a validation function that will return the information and throw errors if it's not in the .env. Example - 
```/**
 * @returns {EnvInfo}
 */
function validateEnvInfo() {
  if (typeof process.env.EMAIL === 'undefined') {
    throw new Error('Must supply EMAIL in .env');
  }
  if (typeof process.env.PASSWORD === 'undefined') {
    throw new Error('Must supply PASSWORD in .env');
  }
  if (typeof process.env.USER_TOKEN === 'undefined') {
    throw new Error('Must supply USER_TOKEN in .env');
  }
  if (typeof process.env.IOT_ADMIN_SYS_KEY === 'undefined') {
    throw new Error('Must supply IOT_ADMIN_SYS_KEY in .env');
  }
  if (typeof process.env.URL === 'undefined') {
    throw new Error('Must supply URL in .env');
  }

  return {
    EMAIL: process.env.EMAIL,
    PASSWORD: process.env.PASSWORD,
    USER_TOKEN: process.env.USER_TOKEN,
    IOT_ADMIN_SYS_KEY: process.env.IOT_ADMIN_SYS_KEY,
    URL: process.env.URL
  };
}
```